src/engraving/libmscore/tremolo.cpp

```diff
@@ -290,7 +290,7 @@ void Tremolo::layoutOneNoteTremolo(qreal x, qreal y, qreal h, qreal spatium)
         yOffset -= up ? 1.5 * spatium : 1 * spatium;
         yOffset -= beams >= 2 ? 0.5 * spatium : 0.0;
     } else if (beams) {
-        yOffset -= beams * 0.75 * spatium;
+        yOffset -= beams * (score()->styleD(Sid::beamDistance) == 1.0 ? 1.0 : 0.75) * spatium;
```

Comment:
Would it eventually make sense to change the style value to a bool like "useWideBeams", and do the conversion for old scores while importing those? 
Otherwise, it could be good to use `qFuzzyIsNull(score()->styleD(Sid::beamDistance))` instead of a plain comparison.

Reply:
I would thing a `useWideBeams` style parameter would make much more sense—especially since I've made that comparison in several places in this PR.

I did notice that the `beamDistance` parameter is used in `src/engraving/libmscore/scorefont.cpp` on line 531. I'm not as familiar with that part of the codebase yet. Do you know if updating the style parameter to be a boolean would negatively affect the `scorefont.cpp` code?

If not, I'll go ahead and update the style parameter in this PR as well.

Reply:
Ok, I believe I updated everything correctly including the UI code. Could you double-check it? I've not touched any UI code yet, so while it seems to be working, you probably have a better idea about all the edge cases.

Reply:
Looks good to me!